src/Sdk/DTWebApi/WebApi/TimelineRecord.cs

```diff
@@ -209,35 +217,46 @@ public TimelineReference Details
             set;
         }
 
-        [DataMember(Order = 40)]
-        public Int32? ErrorCount
+        public Int32 ErrorCount
         {
-            get;
-            set;
+            get
+            {
+                return m_errorCount.GetValueOrDefault(0);
+            }
+            set
+            {
+                m_errorCount = value;
+            }
         }
```

Comment:
I'm not sure why these were nullable in the first place.  I'm guessing it was for backwards compatibility with older serialized versions that lacked these fields?

Making them non-nullable has some compelling advantages and eliminates subtle bugs like not having to worry about the impact null might have on everyday operators like `++`, `<`, `>`, etc.

see https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types#lifted-operators